proto/osmosis/concentratedliquidity/v1beta1/query.proto

```diff
@@ -2,8 +2,8 @@ syntax = "proto3";
 package osmosis.concentratedliquidity.v1beta1;
 
 import "gogoproto/gogo.proto";
-import "osmosis/concentrated-liquidity/params.proto";
-import "osmosis/concentrated-liquidity/tickInfo.proto";
+import "osmosis/concentratedliquidity/params.proto";
+import "osmosis/concentratedliquidity/v1beta1/tickInfo.proto";
```

Comment:
why did we add v1beta1 here, can we undo this part?

Reply:
same here https://github.com/osmosis-labs/osmosis/pull/6413#discussion_r1383630059